CHANGELOG.asciidoc

```diff
@@ -21,11 +21,16 @@ endif::[]
 [[unreleased]]
 === Unreleased
 
-https://github.com/elastic/apm-agent-go/compare/v2.4.6...main[View commits]
+https://github.com/elastic/apm-agent-go/compare/v2.4.7...main[View commits]
 
 [[release-notes-2.x]]
 === Go Agent version 2.x
 
+[[release-notes-2.4.7]]
+==== 2.4.7 - 2023/11/23
+
+- Bump submodule dependency version {pull}1546[#1546]
```

Comment:
nit: if there was an issue with 2.4.6 I'd prefer that we mention it, either under 2.4.7 or 2.4.6.